packages/signature-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [22.0.0]
+
+### Added
+
+- Add isDecodeSignatureRequestEnabled to SignatureController constructor to be able to check if decoding api should be invoked for the signature ([#4903](https://github.com/MetaMask/core/pull/4903))
+- Integrating signature decoding api ([#4855](https://github.com/MetaMask/core/pull/4855))
```

Comment:
This is implied by the other entries, but we can add more detail in a nested bullet point under one of the others?

Reply:
This is not clear to me, so we keep only 1 entry here ?

Reply:
I'm suggesting we have two entries as it spans two PRs, but keep them focused on the API changes.

```
- Add `isDecodeSignatureRequestEnabled`...
- Add `decodingApiUrl` property to specify URL of API to provide additional decoding data.

Reply:
Done 👍